test-app/src/main/java/org/readium/r2/testapp/reader/EpubReaderFragment.kt

```diff
@@ -175,19 +151,20 @@ class EpubReaderFragment : VisualReaderFragment(), EpubNavigatorFragment.Listene
 
     override fun onCreateOptionsMenu(menu: Menu, menuInflater: MenuInflater) {
         super.onCreateOptionsMenu(menu, menuInflater)
-        menuInflater.inflate(R.menu.menu_epub, menu)
```

Comment:
All the options menu items are now in a single layout inflated by `BaseReaderFragment`.